Makefile

```diff
@@ -42,7 +42,7 @@ generate: build # Generate CLI docs and samples
 	go generate ./...
 
 .PHONY: bindata
-OLM_VERSIONS = 0.16.1 0.17.0 0.18.1
+OLM_VERSIONS = 0.16.1 0.17.0 0.18.2
```

Comment:
This must be done in a separate PR prior to a release PR

Reply:
https://github.com/operator-framework/operator-sdk/pull/4995 - Separate PR for OLM version changes.

Reply:
PR merged.